files/default/rsync_ca_cwlogs.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+set -e
+
+# get the list of ca hosts
+ca_list=`curl -s -u $CA_STATUS_USER_PASS $CA_STATUS_URL | jq -r '.[].address'`
+ca_count=`echo $ca_list | wc -w`
+echo "Fetched $ca_count entries"
+
+# rsync the logs
+for ca_host in $ca_list; do
+    echo "fetching from $ca_host"
+    rsync -e "ssh -i $CA_KEY_FILE" -azvh --quiet --inplace \
```

Comment:
This `--inplace` flag is important as it tells rsync to update the existing, local file vs replace with the new one. Without it the behavior of the cw log agent is unpredictable.